packages/apollo-server-koa/package.json

```diff
@@ -27,12 +27,6 @@
   "dependencies": {
     "@apollographql/graphql-playground-html": "1.6.24",
     "@koa/cors": "^2.2.1",
-    "@types/accepts": "^1.3.5",
-    "@types/cors": "^2.8.4",
-    "@types/koa": "^2.0.46",
-    "@types/koa-bodyparser": "^4.2.1",
-    "@types/koa-compose": "^3.2.2",
-    "@types/koa__cors": "^2.2.1",
```

Comment:
At the very least, `koa` is used on the exported `KoaHandler` as `Koa.Context`:

https://github.com/apollographql/apollo-server/blob/aa3e23de9fbcde41a1bd3cca9c2eacf734bae582/packages/apollo-server-koa/src/koaApollo.ts#L1-L16